### DISCUSSION TITLE
Change A User's Email with Web3Auth's Email Auth

### DISCUSSION THREAD

**Discussion by ajmontgomery88** [Web3Auth/Web3Auth#810] [Help]
We are using Web3Auth's email authentication for the Solana Torus Wallet in our mobile application.

In the mobile app, the user can change their email, so how can we reflect this change within the Torus Wallet?

Is there a process to update the email and verify the email after updating?

Can this be done with the following API or within Web3Auth's Dashboard?

https://docs.tor.us/solana-wallet/api-reference/account

Please help, thanks!

**Comment by YZhenY** [comment]
This is not possible with the Torus Wallet, you can if you use Web3Auth and customauth by keeping the users uniqueID consistant

**Reply by ajmontgomery88** [reply]
Thank you! We planned on using a uniqueID, and we will need to set up Customauth using the following Web3Auth prompt?

How, can we change the email? Via API or using the Web3Auth Dashboard?

Thanks!

![Web3Auth Screenshot](https://user-images.githubusercontent.com/110572313/199540996-8cec61dc-a87a-45b3-8f89-49f4d33fbe7c.png)
